build-script.py

```diff
@@ -730,7 +730,9 @@ def main():
             printerr(
                 "FAIL: Gyb-generated files committed to repository do "
                 "not match generated ones. Please re-generate the "
-                "gyb-files and recommit them."
+                "gyb-files using the following command, open a PR to the "
+                "SwiftSyntax project and merge it alongside the main PR."
+                "$ swift-syntax/build-script.py --degyb-only --toolchain whatever"
```

Comment:
Not sure if there's a brief way to say this, but it could be worth noting that it doesn't need to be a toolchain. Ie. `/path/to/swift-build/` will generally work.

Reply:
What do you think about the following?
```
swift-syntax/build-script.py --degyb-only --toolchain /dev/null
```